cmd/api/main.go

```diff
@@ -269,10 +269,11 @@ func createChainIDStack(
 		return chains.ChainStack{}, fmt.Errorf("starting event processor: %s", err)
 	}
 	return chains.ChainStack{
-		Store:                 systemStore,
-		Registry:              registry,
-		EventProcessor:        ep,
-		AllowTransactionRelay: config.AllowTransactionRelay,
+		Store:          systemStore,
+		Registry:       registry,
+		EventProcessor: ep,
+		// TODO: we can remove the AllowTransactionRelay config property entirely in a future PR
+		AllowTransactionRelay: false,
```

Comment:
This is redundant since all of the config files have been changed, but I don't see any harm in doing this.